web/sites/default/config/default/field.field.node.affinity_group.field_use_ext_email_list.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 272aead9-cf82-49af-8007-df82f1ca367a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_use_ext_email_list
+    - node.type.affinity_group
+id: node.affinity_group.field_use_ext_email_list
+field_name: field_use_ext_email_list
+entity_type: node
+bundle: affinity_group
+label: 'Use External Email List'
+description: "Check to enable use of Simple Lists email list.\r\nAn email list will be created with using field_group_slug for the address."
```

Comment:
Let's use the text "Check to enable a discussion email list. The email list will be created using the Slug for the address".